app/handlers/analysis.py

```diff
@@ -55,14 +55,16 @@ def post(self):
     def handler(identifier, email):
         project_path = get_project_path(identifier)
         if not os.path.exists(project_path):
-           return (500, 'Project directory does not exist. Check your identifier?')
+            StatusHelper.set_status(identifier, Status.Type.OBJECT_TRACKING, Status.Flag.FAILURE)
+            return (500, 'Project directory does not exist. Check your identifier?')
 
         status_code, reason = ObjectTrackingHandler.handler(identifier, email, AnalysisHandler.object_tracking_callback)
         return (status_code, reason)
 
     @staticmethod
     def object_tracking_callback(status_code, response_message, identifier, email):
         if status_code == 200:
+            StatusHelper.set_status(identifier, Status.Type.SAFETY_ANALYSIS, Status.Flag.IN_PROGRESS)
```

Comment:
This is unnecessary but okay, since we set the in_progress flag in the prepare.  If there was an else that set it to failure when the status_code is not 200, that might be useful.